### ISSUE TITLE
[Preview version] Sorting bug with renamed manga

### ISSUE THREAD

**Issue by Quickdesh** [jobobby04/TachiyomiSY#485]
### Steps to reproduce

1. Add a manga to Library
2. Change Manga name
3. Sort library by "unread"

### Expected behavior

What should happen is that the manga which have been read (library is sorted by "unread"), are to be sorted "alphabetically", using the new name that has been given to the manga.

Example:- 
like how it is in j2k

I renamed
"Teasing master Takagi San" 
to 
"Karakai Jouzu no Takagi-san"

![Screenshot_20211230-171023](https://user-images.githubusercontent.com/36792807/147735684-f5e42259-d9ee-4da0-b2d0-870220401a9d.png)

### Actual behavior

What instead happens is that the "alphabetical" sorting, takes the original name and sorts the library, rather than the updated/renamed name. It sorts with the changed name when the sort is specified as "alphabetical". This bug only arises when sort is "unread".

Example:-

I renamed
"Teasing master Takagi San" 
to 
"Karakai Jouzu no Takagi-san"

![Screenshot_20211230-154659](https://user-images.githubusercontent.com/36792807/147735341-20b489fc-58c1-4278-9753-96cd4e48e296.png)

TL;DR updated Manga name doesn't move alphabetically when sort is in "Unread" mode. It uses the original name rather than the renamed name.

### Crash logs

No Crash takes place

### Tachiyomi version

Preview r423 , TachiyomiSY

### Android version

Android 11

### Device

Motorola edge 20

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.7.0](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by jobobby04** [comment]
Should be fixed in the dev builds

**Comment by Quickdesh** [comment]
Almost a year later... Let's gooo!!
